.github/workflows/lint.yml

```diff
@@ -46,4 +46,4 @@ jobs:
       
       - name: Run Linters
         run: |
-          yarn check-format
+          yarn format
```

Comment:
I think we should stick with `check-format` here and have the CI only throw an error when code is not formatted properly. If we run `format` here, basically it will format the code for the CI, but not commit it, so our codebase will remain unformatted I think

Reply:
oh i see yes